NoNicotine_Business/Handler/CreatePatientCommandHandler.cs

```diff
@@ -5,6 +5,7 @@
 using Microsoft.Extensions.Configuration;
 using Microsoft.Extensions.Logging;
 using NoNicotine_Business.Commands;
+using NoNicotine_Business.Repositories;
```

Comment:
All of this changes shouldn't be merged with the master branch (they're exclusively for testing purposes)